maxim/models/query_builder.py

```
@@ -82,9 +82,9 @@ def deployment_var(self, key: str, value: Union[str, int, bool], enforce: bool =
             QueryBuilder: The current QueryBuilder instance for method chaining.
         """
 
-        if len(self.query)>0:
+        if len(self.query) > 0:
             self.query += ","
-        self.query += f"{'!!' if enforce else ''}{key}={value}"
+        self.query += f"{'!!' if enforce else ''}{key}={json.dumps(value) if isinstance(value, list) else value}"
```

Pylint (3.3.7)

[convention] Line too long (113/100)

(C0301)